ledger/sandbox/src/main/resources/db/migration/h2database/V20__Pruned_offset.sql

```diff
@@ -0,0 +1,12 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V20: Pruned offset tracking
+--
+-- This schema version adds tracking of the pruned portion of the participant ledger to the
+-- parameters table.
+---------------------------------------------------------------------------------------------------
+
+-- Add the column for most recent pruning offset to parameters.
+ALTER TABLE parameters ADD COLUMN participant_pruned_up_to_inclusive bytea;
```

Comment:
I think you want [`BLOB`](http://h2database.com/html/datatypes.html#blob_type) here, not `BYTEA`. That's a PostgreSQL thing. It might work in H2 but it's not canonical.

Reply:
Good catch - I actually want the same type as `parameters.ledger_end` which for h2 is `BINARY`.